app/controller.go

```diff
@@ -81,6 +81,13 @@ func (c *Controller) Handler() http.Handler {
 	return router
 }
 
+func (c *Controller) createHostHTTPHandler() HTTPHandler {
+	if c.opsConfig.CreateHostDisabled {
+		return notAllowedHttpHandler()
```

Comment:
nit: Instead of `notAllowedHttpHandler` being a function that creates and returns an http handler it could be the handler itself and be returned here:
```
return notAllowedHttpHandler // without the () at the end

...

func notAllowedHttpHandler(w http.ResponseWriter, r *http.Request) error {
    return NewMethodNotAllowedError("Operation is disabled", nil)
}
```

Reply:
yes indeed, good catch! thanks